packages/console/src/pages/Connectors/components/Guide/index.tsx

```diff
@@ -52,12 +54,13 @@ const Guide = ({ connector, onClose }: Props) => {
     watch,
   } = methods;
 
-  const onSubmit = handleSubmit(async (data) => {
-    if (isSubmitting) {
+  const parseJsonConfig = (config: string) => {
```

Comment:
this function is used in both creating and editing connectors. Should we move this method to utils, and hence we can reuse it instead of defining it every time?

Reply:
updated